src/rust/src/libccxr_exports/mod.rs

```diff
@@ -1 +1,32 @@
 //! Provides C-FFI functions that are direct equivalent of functions available in C.
+
+use crate::ccx_options;
+use lib_ccxr::util::log::*;
+use std::convert::TryInto;
+
+/// Initializes the logger at the rust side.
+///
+/// # Safety
+///
+/// `ccx_options` in C must initialized properly before calling this function.
+#[no_mangle]
+pub unsafe extern "C" fn ccxr_init_basic_logger() {
+    let debug_mask =
+        DebugMessageFlag::from_bits(ccx_options.debug_mask.try_into().unwrap()).unwrap();
+    let debug_mask_on_debug =
+        DebugMessageFlag::from_bits(ccx_options.debug_mask_on_debug.try_into().unwrap()).unwrap();
```

Comment:
In both lines, the first `unwrap` is for converting a `LLONG` to an `u16`. If this fails, the value is invalid anyway and it is better to crash the program. The second one converts u16 to a bitflag. Again if this fails, the value is invalid and it is better to crash the program.

What is the meaning of proper error handling? Because we definitely cannot use `Result` here. We cannot even use `fatal!`, since logger is not yet created. The only thing I can think of is to replace `unwrap` with `expect`. Is this what you want me to do?

Reply:
I have converted the `unwrap` to `expect`.